README.md

```diff
@@ -1,2 +1,148 @@
-# AndroidKotlinMVVMTemplate
-Our own Android template for new projects
+[![Kotlin Version](https://img.shields.io/badge/kotlin-1.6.10-blue.svg)](http://kotlinlang.org/)
+[![Gradle](https://img.shields.io/badge/gradle-7.4-blue.svg)](https://lv.binarybabel.org/catalog/gradle/latest)
+[![API](https://img.shields.io/badge/API-23%2B-blue.svg)](https://android-arsenal.com/api?level=23)
+[![License](https://img.shields.io/badge/License-MIT-lightgrey.svg)](https://en.wikipedia.org/wiki/MIT_License)
+
+# Tapptitude's Android Template
+
+A 100% Kotlin-based project template that helps us kick start our Android projects.
+
+## Main features
+
+- Dependencies managed through Gradle version catalogs `*.toml`
```

Comment:
Do you think a hyperlink to docs ([https://docs.gradle.org/current/userguide/platforms.html#sub:version-catalog]) would be helpful?

Reply:
Great idea ;)